src/SciAssist/utils/windows_pdf2text.py

```diff
@@ -19,7 +19,8 @@ def process_pdf(path):
     for page_layout in extract_pages(path):
         for element in page_layout:
             if "get_text" in dir(element):
-                text = element.get_text().replace("\n","")
+                text = element.get_text().replace("-\n","") # Removes hyphen from a word split between lines
+                text = text.replace("\n"," ") # Replace "\n" with space (between words)
```

Comment:
change back to the original code, designed to strip all spaces in the string.

Later, all segments/lines are joined with " ".